run-utils.mjs

```diff
@@ -52,7 +52,6 @@ function make_compiler(compiler_bytes) {
     const schism = await engine.loadWasmModule(await compiler_bytes());
     engine.setCurrentInputPort(bytes);
     let module_package = schism.exports['compile-stdin->module-package']();
-    module_package = engine.collect(module_package);
```

Comment:
Since we have a GC now, we don't need this split between the two halves of the compiler anymore. The calls to `compile-stdin->module-package` and `compile-module-package->stdout` can be replaced with one call to `compile-stdin->stdout`.

I'm fine with leaving this for later though.